pkg/mls/store/store.go

```diff
@@ -605,3 +605,20 @@ func (s *Store) RunInTx(
 	done = true
 	return tx.Commit()
 }
+
+func (s *Store) RunInSerializableTx(ctx context.Context, numRetries int, fn func(ctx context.Context, txQueries *queries.Queries) error) error {
+	var err error
+	for i := 0; i < numRetries; i++ {
+		select {
+		case <-ctx.Done():
```

Comment:
noob Q: What does this select on this ctx do?

The default case returns nil when it finishes, but ctx seems like some sort of global context? When would it trigger?

Reply:
If you `ctrl+c` then `ctx.Done()` channel will receive a struct and this `select` block will abort.

Reply:
Exactly. It just stops you from continuing retries when the server is in the process of shutting down